install/advanced/core/index.rst

```diff
@@ -2214,8 +2214,8 @@ Logout and login again on shell and then execute:
 
   docker run -it hello-world
 
-4. Deploy a vanilla GeoNode 3.2.0 with Docker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+4. Deploy a vanilla GeoNode with Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Can you align the legnth of the underline "^^^^^" with the title one?

Reply:
I cannot save the commits. ![image](https://user-images.githubusercontent.com/23359514/171220232-0b98f388-a0a7-496c-8e60-bf62990aadcf.png)